inc/user.class.php

```diff
@@ -984,6 +984,13 @@ function applyRightRules() {
                   $right->add($retr_profile);
                }
             }
+			
+			if (isset($this->input["groups_id"])) {
+               $group_user = new Group_user();
+               $input['users_id'] = $users_id;
+               $input['groups_id'] = $this->input["groups_id"];
+               $group_user->add($input);
```

Comment:
The new relation have to be set as dynamic, to be clean when rules change.

But this raise the issue as dynamic group are clean during ldap sync...